src/style.css

```diff
@@ -0,0 +1,29 @@
+* {
+  margin: 0;
+}
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document